scripts/install

```diff
@@ -242,37 +242,37 @@ ynh_script_progression --message="Installing Etherpad plugins..." --weight=90
 
 pushd "$final_path"
 # Add Left/Center/Right/Justify to lines of text in a pad
-npm install ep_align >> $install_log 2>&1
+npm install ep_align@${ep_align_version} >> $install_log 2>&1
 # Framapad - Adds author names to span titles
-npm install ep_author_hover >> $install_log 2>&1
+npm install ep_author_hover@${ep_author_hover_version} >> $install_log 2>&1
 # Automatically disconnects user after some period of time (Prevent server overload)
-npm install ep_automatic_logut >> $install_log 2>&1
+npm install ep_automatic_logut@${ep_automatic_logut_version} >> $install_log 2>&1
 # Framapad - Adds comments on sidebar and link it to the text.
-npm install ep_comments_page >> $install_log 2>&1
+npm install ep_comments_page@${ep_comments_page_version} >> $install_log 2>&1
 # Framapad - Displays paragraphs, sentences, words and characters counts.
-npm install ep_countable >> $install_log 2>&1
+npm install ep_countable@${ep_countable_version} >> $install_log 2>&1
 # Framapad - Delete pads which were never edited
-npm install ep_delete_empty_pads >> $install_log 2>&1
+npm install ep_delete_empty_pads@${ep_delete_empty_pads_version} >> $install_log 2>&1
 # Framapad - Apply colors to fonts
-npm install ep_font_color >> $install_log 2>&1
+npm install ep_font_color@${ep_font_color_version} >> $install_log 2>&1
 # Framapad - Adds heading support to Etherpad Lite.
-npm install ep_headings2 >> $install_log 2>&1
+npm install ep_headings2@${ep_headings2_version} >> $install_log 2>&1
 # Framapad - Edit and Export as Markdown in Etherpad
-npm install ep_markdown >> $install_log 2>&1
+npm install ep_markdown@${ep_markdown_version} >> $install_log 2>&1
 if [ $mypads -eq 1 ]; then
 	# Framapad - Groups and private pads for etherpad
 	npm install ep_mypads@${mypads_version} >> $install_log 2>&1
 fi
 # Framapad - Add support to do 'page view', with a toggle on/off option in Settings, also Page Breaks with Control Enter
-npm install ep_page_view >> $install_log 2>&1
+npm install ep_page_view@${ep_page_view_version} >> $install_log 2>&1
 # Framapad - Add support to do 'Spell checking'
-npm install ep_spellcheck >> $install_log 2>&1
+npm install ep_spellcheck@${ep_spellcheck_version} >> $install_log 2>&1
 # Framapad - Add support for Subscript and Superscript
-npm install ep_subscript_and_superscript >> $install_log 2>&1
+npm install ep_subscript_and_superscript@${ep_subscript_and_superscript_version} >> $install_log 2>&1
 # Framapad - View a table of contents for your pad
-npm install ep_table_of_contents >> $install_log 2>&1
+npm install ep_table_of_contents@${ep_table_of_contents_version} >> $install_log 2>&1
 # Framapad - User Pad Contents font size can be set in settings, this does not effect other peoples views
-npm install ep_user_font_size >> $install_log 2>&1
+npm install ep_user_font_size@${ep_user_font_size_version} >> $install_log 2>&1
```

Comment:
I guess that works, but it really sounds like we're reinventing packages.json here...

Reply:
IIRC Maniack had a lot of problems with the packages.json on this app, and the developer was not very cooperative with him.